en/installation-guide/workstation.rst

```diff
@@ -137,9 +137,12 @@ Install the demo project
 
 * Extract the file
 
-* If you leave the qgep_[language].qm - file in the same directory as the qgep.qgs - file and start QGIS with language, your qgep-project will be translated if you start the project.
+* If you leave the qgep_[language].qm - file in the same directory as the qgep.qgs - file and start QGIS with on of these languages, your qgep-project will be translated to that langue when you start the project. E.g. you start qgep.qgs with a QGIS Installation that is set to German, then the qgep.qgs project will appear in German. 
```

Comment:
```suggestion
* If you leave the qgep_[language].qm files in the same directory as the qgep.qgs file and start QGIS with on of these languages, your qgep-project will be translated to that langue when you start the project. E.g. you start qgep.qgs with a QGIS Installation that is set to German, then the qgep.qgs project will appear in German.

.. note:: You need to explicitly set the language in QGIS in settings. If QGIS is configured to take the system language, the QGEP translation is not loaded.
```

Reply:
It should be "one of these languages, ..." instead of "on of these languages,"
Else it is ok for me.